examples/resource_lacework_alert_channel_victorops/main.tf

```
@@ -1,6 +1,27 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
 provider "lacework" {}
 
 resource "lacework_alert_channel_victorops" "example" {
-  name        = "Victor Ops example"
-  webhook_url = "https://alert.victorops.com/integrations/generic/20131114/alert/31e945ee-5cad-44e7-afb0-97c20ea80dd8/database"
+  name        = var.channel_name
+  webhook_url = var.webhook_url
+  // test_integration input is used in this example only for testing
+  // purposes, it help us avoid sending a "test" request to the
+  // system we are integrating to. In production, this should remain
+  // turned on ("true") which is the default setting
+  test_integration = false
 }
+
+output "channel_name" {
+  value = var.channel_name
```

There is no sense in having an output to point to a variable, we are not testing anything there. 

Point them to the resource:
```suggestion
  value = lacework_alert_channel_victorops.example.name
```

---

cc @vatasha 👆🏽